packages/opentelemetry-core/src/common/ConsoleLogger.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Logger } from '@opentelemetry/types';
+import { LogLevel } from './types';
+
+class ConsoleLogger implements Logger {
+  constructor(level: LogLevel = LogLevel.DEBUG) {
+    if (level >= LogLevel.DEBUG) {
+      this.debug = (...args) => {
```

Comment:
Does this type need to be `(message: string, ...args)`? Seems the TS compiler was OK with it but I would have expected it to need it.

Reply:
Yeah, TS compiler seems ok with this and even if I add no effect. Let me know if you still want me to change.

Reply:
OK, I think it's fine without it then.